app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -54,11 +54,16 @@ def self.raw_connect(hostname, port, options)
   def self.openshift_connect(hostname, port, options)
     require 'openshift_client'
     api_endpoint = raw_api_endpoint(hostname, port)
-    osclient = OpenshiftClient::Client.new(api_endpoint, api_version)
+    ssl_options = {:verify_ssl => OpenSSL::SSL::VERIFY_NONE}
+    auth_options = {}
+    auth_options[bearer_token] = options[:bearer] if options[:bearer]
     # TODO: support real authentication using certificates
-    osclient.ssl_options(:verify_ssl => OpenSSL::SSL::VERIFY_NONE)
-    osclient.bearer_token(options[:bearer]) if options[:bearer]
-    osclient
+    OpenshiftClient::Client.new(
+      api_endpoint,
+      api_version,
+      ssl_options  => ssl_options,
+      auth_options => auth_options,
```

Comment:
@moolitayer these should be:

      ssl_options:  ssl_options,
      auth_options: auth_options